editor/patches/babel-eslint+10.0.1.patch

```diff
@@ -0,0 +1,24 @@
+# generated by patch-package 6.4.7 on 2021-12-17 14:16:25
+#
+# command:
+#   npx patch-package babel-eslint
+#
+# declared package:
+#   babel-eslint: 10.0.1
+#
+diff --git a/node_modules/babel-eslint/lib/analyze-scope.js b/node_modules/babel-eslint/lib/analyze-scope.js
+index 9b2a40d..05b470b 100644
+--- a/node_modules/babel-eslint/lib/analyze-scope.js
++++ b/node_modules/babel-eslint/lib/analyze-scope.js
+@@ -327,11 +327,6 @@ module.exports = function(ast, parserOptions) {
+     fallback,
+   };
+ 
+-  if (OriginalReferencer._babelEslintPatched) {
```

Comment:
what does this fix?

Reply:
it tries to require a non-existing file, before this PR it was 'patched' in our webpack config